webapp/package.json

```diff
@@ -69,7 +69,7 @@
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "~7.9.0",
     "@faker-js/faker": "5.1.0",
-    "@nuxtjs/composition-api": "0.33.1",
+    "@nuxtjs/composition-api": "0.32.0",
```

Comment:
Thanks for your work @ulfgebhardt
I have tried that as well and got a different error messages.

I try again.